crates/orderbook/src/api/get_fee_info.rs

```diff
@@ -39,6 +44,12 @@ pub fn get_fee_info(
     get_fee_info_request().and_then(move |query: Query| {
         let fee_calculator = fee_calculator.clone();
         async move {
+            if query.sell_token == BUY_ETH_ADDRESS {
+                return Result::<_, Infallible>::Ok(
+                    PartialValidationError::InvalidNativeSellToken.into_warp_reply(),
+                );
+            }
```

Comment:
This change is unrelated to this PR, but the legacy (and deprecated) fee endpoint uses a different path for estimating fees than the `quote` and `fee_and_quote` end points.

I snuck it in here, but can move it to a separate PR if you think I should.